docs/source/notebooks/continuous-non-contractual.ipynb

```diff
@@ -17,12 +17,9 @@
    "source": [
     "import pymmmc\n",
     "\n",
-    "import aesara.tensor as at\n",
```

Comment:
Aren’t you missing imports from pytensor here?

Reply:
Not being used, unless I missed something